docs/data/data_locations.md

```diff
@@ -0,0 +1,103 @@
+   A common workflow is for data to be private --> get ingested to LEAP GCS --> OSN Pods for publication.
+
+## LEAP-Pangeo Cloud Buckets
+
+LEAP-Pangeo provides users two cloud buckets to store data. The JupyterHub is automatically authenticated to read from any of these buckets but write access might differ (see below). See [Authentication](../technical-reference/authentication.md) for details on how to access buckets from 'outside' the JupyterHub.
+Google cloud is structured such that it is very easy and cheap to move data *into* the buckets, but there are high egress fees for taking data out of GCP Infrastructure. Taking data out means both writing from GCP to outside and reading from an external source like an HPC.
+!!! tip
+TLDR: Use the LEAP GCS buckets when you are actively doing science using the JupyterHub. If you wish to share some kind of finished product with the world, it is best to "publish" the data by moving outside GCS into OSN.
+
+- `gs://leap-scratch/` - Temporary Storage deleted after 7 days. Use this bucket for testing and storing large intermediate results. leap-scratch is also a great staging area to use while ingesting data to some other permanent location.
+- `gs://leap-persistent/` - Persistent Storage. Use this bucket for storing results you want to share with other members or access consistently from the Hub.
+
+Files stored on each of those buckets can be accessed by any LEAP member, so be conscious in the way you use these.
+
+- **Do not put sensitive information (passwords, keys, personal data) into these buckets!**
+- **When writing to buckets only ever write to your personal folder!** Your personal folder is a combination of the bucketname and your github username (e.g. \`gs://leap-persistent/funky-user/').
+
+GCS is great if:
+\- You want to move data from your Jupyter-Hub home directory to the cloud.
+\- You don't need the data to be accessed outside of the Jupyter-Hub.
+\- This data is a work-in-progress and might be regenerated or modified as you do your science.
+
+## Open Storage Network (OSN) Pod
+
+The [Open Storage Network](https://www.openstoragenetwork.org/) is a federally funded distributed cloud storage for the research community. LEAP has a ~1 petabyte allocation on OSN, which is divided into projects and buckets. Projects are used to give additional users access to the Coldfront Admin Console, whereas buckets are how storage is administered up on the Pod. A project can have multiple buckets.
+
+There are currently 3 principal Projects on the Pod:
+
+- `'leap-pangeo'`: Used for Data Ingestion across the m2lines and LEAP community
+  - Buckets:
+    - `'leap-pangeo-manual'`: **No write access for users**
+    - `'leap-pangeo-pipeline'`: **No write access for users**
+    - `'leap-pangeo-inbox'`: *Write access can be shared with users who want to add data e.g. from an HPC center*
+- `'m2lines'`: Used for project data and publications from the m2lines project
```

Comment:
Maybe we don't need details about m2lines in the LEAP documentation. It would be enough to note that we share the particular OSN pod with them.